### DISCUSSION TITLE
I can't access the repository

### DISCUSSION THREAD

**Discussion by jlgadrow** [tsightler/ring-mqtt#413] [Support Q&A] [answered]
When I try to add the repository in the add-on store, I get the following:

![image](https://user-images.githubusercontent.com/20450601/182030990-28c661f1-fd0d-4368-93c5-c6022fbe58f5.png)

Am I using the wrong URL?

**Comment by tsightler** [comment, marked as answer]
For the addon you need to use the addon repo:

https://github.com/tsightler/ring-mqtt-ha-addon

**Reply by jlgadrow** [reply]
Thank you. I've been pulling my hair for 2 hours!
